src/main/resources/add-ons-to-index.json

```diff
@@ -3605,6 +3710,9 @@
       "uid": "org.openmrs.module.reference-application-module",
       "type": "OMOD",
       "name": "Reference Application Module",
+      "tags": [
+        "reference-application"
```

Comment:
Also add this same tag to:
* Reference Metadata
* Reference Demo Data

Reply:
It was already done by previous commits @djazayeri